app/src/main/res/values/strings.xml

```diff
@@ -825,6 +825,13 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_charging_station_operator_title">"Who is the operator of this charging station?"</string>
     <string name="quest_charging_station_capacity_title">"How many cars can be charged at this charging station at the same time?"</string>
 
+    <string name="team_mode">Team Mode</string>
+    <string name="team_mode_exit">Exit Team Mode</string>
+    <string name="team_mode_description">When mapping the same area together in a group, you can enter team mode to split up quests between all team members.</string>
+    <string name="team_mode_team_size_label">How many people are you?</string>
+    <string name="team_mode_team_size_hint">Team mode works for groups between 2 and 12 people.</string>
+    <string name="team_mode_choose_color">Everybody in the group has to choose a unique color (on their own phone) now.</string>
```

Comment:
It's not just a unique colour now, but maybe unique dot is more confusing?

Reply:
No I think color is better. You suggested yourself to show a boundary in the color chosen.

Reply:
Yeah fair enough.